lib/rucio/core/credential.py

```diff
@@ -107,14 +108,33 @@ def get_signed_url(rse_id, service, operation, url, lifetime=600):
                                                                                  signature)
 
     elif service == 's3':
+
+        # get RSE S3 URL style (path or virtual)
+        # path-style: https://s3.region-code.amazonaws.com/bucket-name/key-name
```

Comment:
The code is good, but I wonder if we should find a better name for the "virtual" variant. Wouldn't "host" be more expressive? As in, in one case the bucket name is in the path and in the other case it's in the host. "virtual" by itself doesn't mean much IMHO. WDYT?

Reply:
Yeah, `host` would make indeed more sense. The reason `virtual` was put there is because that's what the library expects here:
https://github.com/rucio/rucio/pull/5929/files#diff-65d1a04f8f23b2b7967130d235c39b9be0a294a929dcd9b18a6ee8adafd2128dR173
thus the code becomes less verbose.

We can proceed as you see fit, no strong opinions on this.

Reply:
Interesting.... I would prefer `host`, yes, purely for user/operational reasons. But maybe someone else has a stronger opinion? :-D